catalogue_of_life/Loader.java

```diff
@@ -118,6 +118,27 @@ private static void unzip(Path zipFile, Path workingDirectory) {
     }
 
     public static void prepareData(Path dataDirectory) throws IOException {
+        giveNamesUniqueID(dataDirectory);
+        separateTaxonDistributions(dataDirectory);
+    }
+
+    private static void giveNamesUniqueID(Path dataDirectory) throws IOException {
+        TsvParser parser = createTsvParser(dataDirectory.resolve("VernacularName.tsv"));
+        TsvWriter writer = createTsvWriter(dataDirectory.resolve("VernacularNameWithID.tsv"));
+
+        Record record;
+        Map<String, String> recordValues = new HashMap<>();
+        long i = 0;
+        while ((record = parser.parseNextRecord()) != null) {
+            record.fillFieldMap(recordValues);
+            recordValues.put("col:nameID", String.valueOf(i++));
+            writer.writeRow(recordValues);
+        }
+
+        writer.close();
+    }
```

Comment:
Give a unique ID to each vernacular name so that we can unambiguosly create the `naming` relations.